File: src/main/java/com/back/domain/post/post/entity/Post.java
package com.back.domain.post.post.entity;

import com.back.domain.member.member.entity.Member;
import com.back.global.jpa.entity.BaseEntity;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.ManyToOne;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Entity
@Getter
@Setter
@ToString
@NoArgsConstructor
public class Post extends BaseEntity {
    @ManyToOne // Post 가 많고(Many) Member 가 적다(One) // N:1 관계
    private Member author; // 필드명 : AUTHOR_ID, 설명 : MEMBER 테이블의 ID값이 저장
    private String title;
    @Column(columnDefinition = "TEXT")
    private String content;

    public Post(Member author, String title, String content) {
        this.author = author;
        this.title = title;
        this.content = content;
    }
}

File: src/main/java/com/back/domain/post/post/entity/Post.kt
package com.back.domain.post.post.entity

import com.back.domain.member.member.entity.Member
import com.back.global.jpa.entity.BaseEntity
import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.ManyToOne

@Entity
class Post(

    @ManyToOne
    var author: Member? = null,

    var title: String = "",

    @Column(columnDefinition = "TEXT")
    var content: String = ""

) : BaseEntity()